student-work/michael-felsenthal/api-final-project/GET table.md

```diff
@@ -1,6 +1,34 @@
-# GET/tableNo
-Include a description
+# GET Reference Guide
+The order given to the server. Includes the table and order number.  
+
+## Code Guide
+
+Name | type | Req. | Description
+---- | ----- | ----- | --------------------
+Item1 | string | Y |  Describes the customer's order
+Item2 | int  | N | Provides customer's order number and time of order
```

Comment:
Item2 is a second order item not the order number and time

orderNum is the order number
timestamp is the time the order was placed. 
both of these should be in the table